community/planning/queuer_strat_one_round_serial.md

```diff
@@ -0,0 +1,108 @@
+<!--
+  Copyright 2022 Cargill Incorporated
```

Comment:
I think this header should say 2018-2022 instead of just 2022

Reply:
updated, thanks!